logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultPathAssertions.kt

```diff
@@ -109,6 +109,13 @@ class DefaultPathAssertions : PathAssertions {
     override fun <T : Path> isDirectory(container: AssertionContainer<T>): Assertion =
         fileTypeAssertion(container, A_DIRECTORY) { it.isDirectory }
 
+    override fun <T : Path> isRelative(container: AssertionContainer<T>): Assertion =
+        assertionBuilder.descriptive
+            .withTest(container) { !it.isAbsolute }
+            .withPathAttributesFailureHint(container)
+            .withDescriptionAndRepresentation(DescriptionBasic.IS, RELATIVE_PATH)
+            .build()
```

Comment:
note, `isAbsolute()` never throws an IOException it does not interact with the file system. Thus there is no value in 
`withPathAttributesFailureHint(container)` simplify to:
``` assertionBuilder.descriptive
            .withTest(container) { !it.isAbsolute }
            .withDescriptionAndRepresentation(DescriptionBasic.IS, RELATIVE_PATH)
            .build()
```
and for this we have a shortcut (because it is that common):
```suggestion
        container.createDescriptiveAssertion(DescriptionBasic.IS, RELATIVE_PATH) { !it.isAbsolute() }
```

Reply:
btw. sorry for my misleading hint in the description of the issue, was not thinking about it enough when I wrote the issue :disappointed:

Reply:
It was kinda ambiguous, whether this method should perform existing file check or not 😉